src/Adapter/Order/AbstractOrderHandler.php

```diff
@@ -243,6 +245,35 @@ protected function updateSpecificPrice(
         $specificPrice->add();
     }
 
+    /**
+     * @param Number $priceTaxIncluded
+     * @param Number $priceTaxExcluded
+     * @param Order $order
+     * @param Product $product
+     *
+     * @return Number
+     */
+    private function getPrecisePriceTaxExcluded(
+        Number $priceTaxIncluded,
+        Number $priceTaxExcluded,
+        Order $order,
+        Product $product
+    ): Number {
+        $taxAddress = new Address($order->{Configuration::get('PS_TAX_ADDRESS_TYPE', null, null, $order->id_shop)});
```

Comment:
```suggestion
        $taxAddress = new Address($order->{Configuration::get('PS_TAX_ADDRESS_TYPE', null, null, $order->id_shop)});
```

You don't check the return of `Configuration::get` exists in $order ?

Reply:
Nope, this is a mandatory configuration that is installed with the Shop If it doesn't exist the whole checkout will be broken so this part won't be the most important ^^

And I couldn't use the `ConfigurationInterface` because it's an abstract class so it would force all classes that use this method to be injected the service even if they don't use it nor this method

Reply:
Ok for me.